packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/talon/clearKey3.yml

```diff
@@ -0,0 +1,30 @@
+languageId: talon
+command:
+  version: 5
+  spokenForm: clear key
+  action: {name: clearAndSetSelection}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: collectionKey}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |-
+    tag: user.aaa
+    and not tag: user.bbb
+    -
+    aaa: bbb
+  selections:
+    - anchor: {line: 1, character: 21}
+      active: {line: 1, character: 21}
+  marks: {}
+finalState:
+  documentContents: |-
+    tag: user.aaa
+    and not : user.bbb
```

Comment:
Question: is this what we want? Not necessarily what I expected, but could be useful, and there is a logic to it. @AndreasArvidsson suggests that maybe "name" should do this and "key" should be everything left of the `:`